lib/utils.js

```diff
@@ -23,7 +23,7 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 var JWT_ISSUER = exports.JWT_ISSUER = 'learnersguild.org';
 
 function idmGraphQLFetch(graphQLParams) {
-  var token = arguments.length <= 1 || arguments[1] === undefined ? null : arguments[1];
+  var token = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : null;
```

Comment:
I didn't make this change explicitly; it must have happened in the rebuild from src. Perhaps someone made a change directly to lib previously rather than committing to lib and rebuilding.

@bundacia, @jeffreywescott: please note & advise?